database/src/main/scala/cromwell/database/slick/CallCachingSlickDatabase.scala

```diff
@@ -37,20 +37,12 @@ trait CallCachingSlickDatabase extends CallCachingStore {
     }
   }
 
-  override def fetchCachedResult(metaInfoId: MetaInfoId)(implicit ec: ExecutionContext): Future[Option[CachedResult]] = {
+  override def fetchCachedResult(metaInfoId: MetaInfoId)(implicit ec: ExecutionContext): Future[CachedResult] = {
     val action = for {
-      metaInfo <- dataAccess.metaInfoById(metaInfoId.id).result
+      metaInfo <- dataAccess.metaInfoById(metaInfoId.id).result.head
```

Comment:
I was happy to remove this.  If the metaInfoId was there just a few moments ago in the determination of a cache hit, I don't see why it should disappear.